global_map_file: "/home/xiangweizeng/my_project/fast-lio-relocalization_ws/src/FAST_LIO_LOCALIZATION/PCD/meeting-room/GlobalMap.pcd"

rough_scan_resolution: 0.25
rough_map_resolution: 0.25
rough_max_iteration: 5
rough_score_thresh: 0.2

refine_scan_resolution: 0.1
refine_map_resolution: 0.1
refine_max_iteration: 10
refine_score_thresh: 0.1

t_carbody_imu: [ 0., 0., 0. ]
r_carbody_imu: [ 1., 0., 0.,
                   0., 1., 0.,
                   0., 0., 1.]